### PR TITLE
Fix SEQUREH7V2

### PR DESCRIPTION
**Pull request by swagprincess**
- Fix motor 1 (PB4) not being assigned in timer_pin_map.
- Change the DMA field from -1 to 0 in timer_pin_map as per H7 guidelines (DMA arbiter).
- Remove unnecessary gyro drivers (difference between V1 and V2 is just the gyro).
- Add DMA definitions for TIMUP2 and 4, used for motors 5-8, for burst dshot.
- Remove unnecessary TIMUP definitions and I2C pins from timer_pin_map.
- General restructuring/legibility.

I have flown this config in my build various times and have had no problems.

### CODE REVIEW

File: configs/SEQUREH7V2/config.h [hunk 1]
```diff
@@ -111,15 +133,22 @@
...
+
 #define BEEPER_INVERTED
+
 #define MAX7456_SPI_INSTANCE SPI1
+
 #define GYRO_1_SPI_INSTANCE SPI2
-#define FLASH_SPI_INSTANCE SPI3
 #define GYRO_1_ALIGN CW90_DEG
+
+#define FLASH_SPI_INSTANCE SPI3
```
Comment (haslinghuis) [inline configs/SEQUREH7V2/config.h:154]: - No new line on EOF
```suggestion
#define FLASH_SPI_INSTANCE SPI3

```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by swagprincess** [comment]
Thank you! I've seen in other targets camera control being left to 0 so I assumed it was okay

**Comment by swagprincess** [comment]
somehow managed to split that into 2 commits, sigh, should be okay now

File: configs/SEQUREH7V2/config.h [hunk 2]
```diff
@@ -111,15 +135,22 @@
...
```
Comment (haslinghuis) [inline configs/SEQUREH7V2/config.h:156]:
```suggestion
#define FLASH_SPI_INSTANCE SPI3

```

**Review by haslinghuis — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Pull request merged**